src/bin/cratesfyi.rs

```diff
@@ -239,6 +240,13 @@ pub fn main() {
         } else if let Some(_) = matches.subcommand_matches("update-search-index") {
             let conn = db::connect_db().unwrap();
             db::update_search_index(&conn).expect("Failed to update search index");
+        } else if let Some(_) = matches.subcommand_matches("move-to-s3") {
+            let conn = db::connect_db().unwrap();
+            let mut count = 1;
+            while count != 0 {
+                count = db::file::move_to_s3(&conn, 5_000).expect("Failed to update search index");
```

Comment:
The `expect` message didn't get updated. It should probably be something like "Failed to upload batch to S3".

Reply:
Ah, yes, indeed. Fixed now -- I also added a `total` so there's a relatively easy way to tell how many we've uploaded so far without looking at relatively slow queries that search the whole file table.